internal/stores/s3.go

```diff
@@ -130,45 +127,27 @@ func (s *S3Store) GetOptions() Options {
 	return s.Options
 }
 
+func (s *S3Store) GetFsys() afero.Fs {
+	return s.fsys
+}
+
 // TODO(discentem): #34 largely copy-pasted from stores/local/local.go. Can be consolidated
 // Upload generates the metadata, writes it to disk and uploads the file to the S3 bucket
 func (s *S3Store) Upload(ctx context.Context, objects ...string) error {
-	if s.Options.MetadataFileExtension == "" {
-		return ErrMetadataFileExtensionEmpty
-	}
-
 	for _, o := range objects {
-		logger.V(2).Infof("object: %s", o)
-		f, err := s.fsys.Open(o)
+		fi, err := s.fsys.Open(o)
```

Comment:
what is `fi` instead of `f` here? Maybe we should spell it out if its different from just `f`.